airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -162,23 +171,44 @@ class Config:
         default_factory=lambda: datetime.now(tz=timezone.utc),
     )
 
-    include_deleted: bool = Field(
-        title="Include Deleted Campaigns, Ads, and AdSets",
+    campaign_statuses: Optional[List[ValidCampaignStatuses]] = Field(
+        title="Campaign Statuses",
         order=4,
-        default=False,
-        description="Set to active if you want to include data from deleted Campaigns, Ads, and AdSets.",
+        description="Select statuses you want to to be loaded in the stream. Empty means all available.",
```

Comment:
Need change description to new default version. 
```suggestion
        description="Select statuses you want to to be loaded in the stream. All means all available.",
```